spec.bs

```diff
@@ -2502,6 +2512,9 @@ To <dfn>send report</dfn> given a [=URL=] |url|:
     :   [=request/redirect mode=]
     :: "`error`"
 
+    Issue: One of the side-effects of a `null` client for this subresource request is it neuters
+    all service worker interceptions, despite not having to set the service workers mode.

```

Comment:
...Maybe we should just set it to "no service workers" explicitly?

Reply:
IIRC, we didn't set service workers mode explicitly in Chrome's network code, that's why we had this note. But just setting it to no service workers is probably a good idea.

Reply:
Agreed. Not setting it in the implementation still gives it a default value under the hood. I'd make that explicit in the spec.

Reply:
This was at the bottom of my inbox, but it looks like this comment was never addressed, right? We did not set the service workers mode explicitly, is that right?

Reply:
sorry forgot that. Here it is: https://github.com/WICG/turtledove/pull/1351.